scripts/asset-import/cli.js

```diff
@@ -16,6 +16,10 @@ const { argv } = yargs
         alias: 'b',
         description: 'The branch or tag to use',
     })
+    .option('assetDirectory', {
+        alias: 'a',
+        description: 'Directory from which to take assets',
+    })
     .option('working-dir', {
```

Comment:
nit: the other options are kebab cased

```suggestion
    .option('asset-dir', {
        alias: 'a',
        description: 'Directory from which to take assets',
    })
    .option('working-dir', {
```